packages/atomic/src/components/commerce/atomic-commerce-search-box/atomic-commerce-search-box.tsx

```diff
@@ -650,6 +654,16 @@ export class AtomicCommerceSearchBox
       : this.bindings.i18n.t('query-suggestions-unavailable');
   }
 
+  private updateQueryWithoutQuerySuggestionTrigger(query: string) {
```

Comment:
An alternative would be to add an optional `fetchQuerySuggestions` parameter (which would default to `true`) on the search box controller's `updateText` method.

Reply:
I do not like this alternative: It adds complexity into a (IMO) perfectly fine function.

Dispatching actions directly is what we recommend implementers to do when a controller does not fit exactly their needs: just applying what we preach ;)